src/test/bash/primedb.sh

```diff
@@ -8,8 +8,8 @@ if [[ $1 == "-h" || $1 == "--help" ]]; then
 Usage: $0
 
 Required Environment Variables:
-    EXCHANGE_ROOTPW: Exchange root password (Must be similar to what is set in exchange's config.json)
-
+     EXCHANGE_ROOTPW: Exchange root password (Must be similar to what is set in exchange's config.json)
```

Comment:
I don't think you need to make any changes in `primedb.sh` so you should probably get rid of these.